client/src/api/users.js

```diff
@@ -1,7 +1,7 @@
 import httpClient from './httpClient';
 
 export const getUser = (id) => {
-  return httpClient.get(`/users/${id}`);
+  return httpClient.get(`/api/user/${id}`);
```

Comment:
restful show routes generally use the "things/:id" format, but maybe I'm not understanding the route's purpose.

Reply:
I agree, same way that the projects api is set up as api/projects/

Reply:
@tsyaeger Just trying to get user from backend, I guess I just thought backend routes start with /api, but I have no issues with /users/${id}